domains/techquity.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "doflamin9o",
+    "email": "coolguys.dev@gmail.com"
+  },
+  "records": {
+    "CNAME": "doflamin9o.github.io"
```

Comment:
No fields properly filled out inside the pull request, no description, no preview, and 404 CNAME. Denied.